cluster/common.sh

```diff
@@ -160,6 +160,28 @@ function clear-kubeconfig() {
   echo "Cleared config for ${CONTEXT} from ${KUBECONFIG}"
 }
 
+# Creates a kubeconfig file for the kubelet.
+# Args: address (e.g. "http://localhost:8080"), destination dir
+function create-kubelet-kubeconfig() {
```

Comment:
Originally it seemed appropriate because there were so many other create-kubeconfig-for-x functions in common.sh, but in retrospect, that's a good point.